features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js

```diff
@@ -20,16 +20,30 @@
  * This backendServiceInvoker contains the wrappers for back end jaggary calls.
  */
 var backendServiceInvoker = function () {
+    var log = new Log("/app/modules/backend-service-invoker.js")
     var publicXMLHTTPInvokers = {};
     var publicWSInvokers = {};
     var publicHTTPClientInvokers = {};
     var IS_OAUTH_ENABLED = true;
```

Comment:
Moved to constant in commit https://github.com/wso2/carbon-device-mgt/commit/3e537e2d841ca3c7946cd34e9da96f261e0ac94e1